scripts/macosx_build.sh

```diff
@@ -1,14 +1,44 @@
 #!/bin/bash
 
+echo `pwd`
+
 # Dependencies
 python3 scripts/tooltool.py --url http://taskcluster/tooltool.mozilla-releng.net fetch -m macos/qt-dependencies.tt.manifest
 export PATH="`pwd`/qt/bin:$PATH"
-# TODO: internal pypi mirror, if it doesn't get used by default?
-pip3 install glean_parser==3.5
+export PATH="/Users/task_163336061065616/Library/Python/3.6/bin:/Users/task_163336061065616/.gem/ruby/2.6.0/bin:$PATH"
```

Comment:
I don't think `task_163336061065616` id is a stable path. Once this runs on a level-3 queue (or another mac ), i would expect this to be different.
I guess you could use `${realpath ~Library/Python/3.6/bin}:$PATH`?